### PR TITLE
Gradle 6.0 compatibility fixes

### PR DESCRIPTION
**Pull request by ljacomet**
Resolve issues related to Gradle 6.0 compatibility

### CODE REVIEW

File: src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerLogsContainer.groovy [hunk 1]
```diff
@@ -97,6 +97,11 @@ class DockerLogsContainer extends DockerExistingContainer {
     @Optional
     Writer sink
 
+    // Allows subclasses to carry their own logic
+    protected Date getInternalSince() {
+        return since.getOrNull();
```
Comment (bmuschko) [inline src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerLogsContainer.groovy]: We can clean it up but let's remove the comma at the end.

File: src/main/groovy/com/bmuschko/gradle/docker/utils/IOUtils.groovy [hunk 2]
```diff
@@ -44,6 +44,5 @@ final class IOUtils {
         ProgressLoggerFactory factory = registry.get(ProgressLoggerFactory)
         ProgressLogger progressLogger = factory.newOperation(Objects.requireNonNull(clazz))
         progressLogger.setDescription("ProgressLogger for ${clazz.getSimpleName()}")
-        progressLogger.setLoggingHeader(null)
```
Comment (bmuschko) [inline src/main/groovy/com/bmuschko/gradle/docker/utils/IOUtils.groovy]: Will this change make it completely Gradle 6.0-compatible? The creation of a ProgressLogger is still OK?

File: src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerLogsContainer.groovy (hunk 1 above)
Comment (ljacomet) [reply src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerLogsContainer.groovy]: Ah, sorry, too much Java muscle memory 😉

File: src/main/groovy/com/bmuschko/gradle/docker/utils/IOUtils.groovy (hunk 2 above)
Comment (ljacomet) [reply src/main/groovy/com/bmuschko/gradle/docker/utils/IOUtils.groovy]: Yes, only some methods have been removed. See [this commit](https://github.com/gradle/gradle/commit/4ca2ff6c4e8f1bb406dcc9a051899256a2e3586e).

Comment (cdancy) [reply src/main/groovy/com/bmuschko/gradle/docker/utils/IOUtils.groovy]: So technically it's still there just the setter is removed. In theory we should be able to do:
```
progressLogger.loggingHeader = null
```
Though that seems a redundant operation. Just putting it out there should folks stumble upon this in the future.

**Comment by bmuschko** [comment]
Thanks for sending in the pull request. It has been merged.

**Pull request merged**